### ISSUE TITLE
WebdriverIO DNS Transfer To CF

### ISSUE THREAD

**Issue by bensternthal** [openjs-foundation/infrastructure#8]
Transfer domains to DNSimple

Transfer certs to Cloudflare (wildcard for all sub domains)

**Comment by bensternthal** [comment]
@christian-bromann Would you be our contact for this?

**Comment by kj-powell** [comment]
@christian-bromann reaching out again. Are you the best contact for this?

**Comment by christian-bromann** [comment]
Yes I would be the project contact. Afaik the `webdriver.io` domain is already owned by the OpenJS Foundation. We currently use AWS to and Route54 to make DNS changes. Happy to move off of it as we plan to get rid of AWS entirely and host on Vercel as other projects already do it. Let me know what information or settings I can provide.

**Comment by kj-powell** [comment]
You are correct- OpenJS does own webdriver.io.
Are there any other domains besides that one?

**Comment by christian-bromann** [comment]
Nope.

**Comment by kj-powell** [comment]
Thanks! We would like to move webdriver.io to DNSimple. Would you be able to provide the configuration needed to do that?

**Comment by christian-bromann** [comment]
I am not sure if I can. While I can update DNS records on AWS I don't have access to the domain configurations as it is owned and handled by the foundation.

**Comment by bensternthal** [comment]
@christian-bromann we own the registration and have the current DNS delegated to AWS. However poking around I do not see webdriver under the foundation's aws account. Is it possible this is managed by the webdriver team?

**Comment by christian-bromann** [comment]
Yes, I am managing the AWS account. In fact it is my personal account and I would love to change that. Is it possible to provide project members access to DNSimple? If I can transfer all DNS records to the new platform I am happy to get rid of AWS all together as we are planning to migrate to Vercel for hosting anyway.

**Comment by bensternthal** [comment]
@christian-bromann agreed on getting this away from your personal account 😸  I do not think we can give you access to DNS simple, however if you are able to export your DNS settings from AWS and send over to raslett@contractor.linuxfoundation.org we can import them and ensure nothing is lost in the transition.

**Comment by christian-bromann** [comment]
> I do not think we can give you access to DNS simple

Mhm 🤔 I am a bit concerned that this creates a bottleneck between the project and the OpenJS Foundation. I often need to tweak DNS entries e.g. to test new integrations for the project or setup e.g. a new sub domain etc. Moving this completely into the hands of the foundation seems not practical as it requires project owners to reach out every time to make a change.

How does the foundation plan to manage this? Is there no chance to enable project members access to these critical infrastructure pieces?

**Comment by bensternthal** [comment]
Moving DNS entries to OpenJS means the Foundation assumes responsibility for maintaining them and addressing any issues that arise. This accountability is the primary reason we don’t grant direct access to the tools we directly manage. DNS is different from something like CloudFlare where we create an account and are then hands off, e.g. you broke it you have to fix it.  We don't have this option with LF's DNS provider currently. 

If you would like the Foundation to take on this responsibility, you’re correct that all changes would need to go through us, typically by submitting issues or requests directly. This choice is entirely up to you and the project.

**Comment by christian-bromann** [comment]
Thanks @bensternthal for the clarification. I would prefer for the WebdriverIO to have the project continue managing the DNS entries themselves. Thank you.

**Comment by bensternthal** [comment]
@christian-bromann understood and no worries. @ryanaslett is there a still a task here to move the certs or is that now moot?

**Comment by bensternthal** [comment]
@ryanaslett will review the above and let us know if we can close this out.

**Comment by bensternthal** [comment]
Closing for now, we can create a new issue if anything changes.
